src/kubernetes/multi_response_decoder.rs

```diff
@@ -0,0 +1,410 @@
+//! Decode multiple [`Response`]s.
+
+use k8s_openapi::{http::StatusCode, Response, ResponseError};
+
+/// Provides an algorithm to parse multiple [`Response`]s from multiple chunks
+/// of data represented as `&[u8]`.
+#[derive(Debug)]
+pub struct MultiResponseDecoder<T> {
```

Comment:
Since this is a single-type module you may  consider doing re-export the value in the `mod.rs` 

```rust
mod multi_response_decoder;
pub use multi_response_decoder::MultiResponseDecoder
```

That will create cleaner end user documentation in `cargo doc` without hampering navigability.